example/index.html

```diff
@@ -175,9 +175,11 @@
       });
       windrose.on("owmlayeradd", windroseAdded, windrose);
     
-    var IndigenousLandsTerritories = L.layerGroup.indigenousLandsTerritoriesLayer();
-    var IndigenousLandsLanguages = L.layerGroup.indigenousLandsLanguagesLayer();
-    var IndigenousLandsTreaties = L.layerGroup.indigenousLandsTreatiesLayer();
+    var IndigenousLandsTerritories = L.layerGroup.indigenousLayers('https://native-land.ca/api/index.php?maps=territories&position=45,-72',"Territories");
```

Comment:
Hey @ananyaarun , why are we passing URL here? I see we are not using it , right? Can we remove this as this will make use of this layer easy ?

Reply:
Oh,so this I passed as the url parameter.
the section of the code where url is defined in options, it is used there.-
`  url: this.url,`
The 3 indigenous layers have different url's.
But now on second thoughts, we can use a similar if else block here as well and avoid another parameter being passed.